syntax = "proto3";

package halfer_service;

message Arguments {
    uint32 argument = 1;
}

message Result {
    uint32 value = 1;
}

service Halfer {
    rpc halfer(Arguments) returns (Result) {}
}
